### DISCUSSION TITLE
Statamic: permissions funky on fresh server, provisioned by dep

### DISCUSSION THREAD

**Discussion by hansipete** [deployphp/deployer#3538] [Help needed]
Hello everybody,

today I spawned a small cloud instance (Ubuntu 20.04) at Hetzner, run `dep init` in my project folder and let the results untouched apart from adding the hostname:
```php
<?php
namespace Deployer;

require 'recipe/statamic.php';

// Config

set('repository', 'git@github.com:hansipote/peak-deployer.git');

add('shared_files', []);
add('shared_dirs', []);
add('writable_dirs', []);

// Hosts

host('hansipote.de')
    ->set('remote_user', 'deployer')
    ->set('deploy_path', '~/peak-deployer');

// Hooks

after('deploy:failed', 'deploy:unlock');
```
Then I provisioned the server by runnning `dep provision -o remote_user=root`. I confirmed all the defaults (blank sudo password) and entered the domain to match the host.

Everything seems to run fine so far. Added content of servers `~/.ssh/id_rsa.pub` as deploy keys on Github.

I'm also able to `dep deploy`, logged into the server and setup the `.env`. The app should be ready to go now, but I'm getting errors:
```
The stream or file "/home/deployer/peak-deployer/releases/2/storage/logs/laravel.log" could not be opened in append mode: Failed to open stream: Permission denied
```
I re-ran the `dep deploy:writable -v` and checked the output and it looks allright (i think) for that folder:
```
[hansipote.de] run cd ~/peak-deployer/current && (getfacl -p storage/logs | grep "^user:www-data:.*w" | wc -l)
[hansipote.de] 1
```

I can manually fix those by `dep ssh` into the server and adjust permission on these folders, but why isn't it working in the first place?

Hope I'm missing something in my procedure and get it running one day :) I'd really like to use it for my deployments.

Thank you & have a great weekend everyone,
Hans

**Comment by antonmedv** [comment]
> but why isn't it working in the first place?

Maybe a bug?)

I recently use Deployer on DO, provision & deploy - everything went well. 

Let's debug more. 

> The stream or file "/home/deployer/peak-deployer/releases/2/storage/logs/laravel.log" could not be opened in append mode: Failed to open stream: Permission denied

 Where error is coming from?

**Reply by hansipete** [reply]
Forgot to mention the context – think it was a 500, output from **Symfony** framework when visiting the site url. (I tinkered a little more so I'm not 100% sure).

If i want to start over on the provisioned server is it fine to `cd ~ && rm -rf *` and then run `dep provision` again?

**Reply by antonmedv** [reply]
I suspect this is php-fpm training to write logs.
What user is used for php-fpm?
